src/CoreFoundation/DispatchData.cs

```diff
@@ -122,6 +134,19 @@ public DispatchData CreateSubrange (nuint offset, nuint size)
 		{
 			return new DispatchData (dispatch_data_create_subrange (Handle, offset, size), owns: true);
 		}
+
+		// copies the dispatch data to a managed array
+		public byte [] ToArray ()
+		{
+			IntPtr bufferAddress = IntPtr.Zero;
+			nuint bufferSize = 0;
+			using DispatchData dataCopy = CreateMap (out bufferAddress, out bufferSize);
+
+			byte[] managedArray = new byte[(int) bufferSize];
```

Comment:
maybe create `bufferSize` as an `int` since it looks like you are casting here and below?

Reply:
Cant be done, since the C API needs it to be a nuint :/